client/my-sites/stats/activity-log/index.jsx

```diff
@@ -472,6 +325,38 @@ class ActivityLog extends Component {
 			</section>
 		);
 
+		const timePeriod = ( () => {
+			let last = null;
+
+			return ( { rewindId } ) => {
+				const ts = 1000 * rewindId;
+
+				if ( null === last ) {
+					last = ts;
+					return (
+						<p key="time-period-head">
+							{ moment( ts ).format(
+								translate( 'MMMM Y', { context: 'moment.js format string to show month and year' } )
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?